packages/fast-components-react-base/src/button/button.props.ts

```diff
@@ -1,11 +1,13 @@
 import * as React from "react";
 import { IButtonClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
 
-export interface IButtonHandledProps {
+export interface IButtonManagedClasses extends IManagedClasses<IButtonClassNameContract> {}
+export interface IButtonUnhandledProps extends React.AllHTMLAttributes<HTMLElement> {}
```

Comment:
Wondering why this couldn't be `HTMLAnchorElement | HTMLButtonElement`

Reply:
At the time the two of those were considered mutually exclusive via TS and would throw a compile error. It may be updated now, but we'd have to look into typescript updates to confirm.